branch_name_verification.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+import robot.basestation.app as app
+from robot.basestation.app import run_shell
+import re
+
+excluded_branches = {'master', 'develop', 'staging', 'test'}
+
+output, error = run_shell("git", "symbolic-ref --short HEAD", False)
+branch_name = output.decode("utf8")[0:-1]
+
+def is_excluded_branch(branch_name, excluded_branches):
+    """
+    check if branch name is in list of excluded branches
+    """
+    return branch_name in excluded_branches
+
+def has_upper_case_letters(trim_issue_num):
+    """
+    check if trimed branch name only contains lower case characters
+    """
+    return not (trim_issue_num.islower())
+
+def has_hyphen_seperator(trim_issue_num):
+    """
+    check for hyphen seperator between branch name and issue number
+    """
+    return trim_issue_num.endswith('-')
```

Comment:
I decided to give this the same treatment as has_upper_case_letters, and just removed it all together. I felt that having this function was useless, because `endswith()` already works on its own